src/main/java/io/kestra/plugin/notifications/sentry/SentryAlert.java

```diff
@@ -149,7 +150,8 @@ public VoidOutput run(RunContext runContext) throws Exception {
             String payload = this.payload != null ? runContext.render(this.payload) : runContext.render(DEFAULT_PAYLOAD.strip());
 
             // Constructing the envelope
-            String envelope = constructEnvelope(payload);
+            var eventId = (String) runContext.getVariables().get("eventId");
```

Comment:
We never have an `eventId` on the `runContext`

Reply:
The [sentry-template.peb ](https://github.com/kestra-io/plugin-notifications/blob/ac4a2f8d18b873c06a764989fa904b9ebc38a7eb/src/main/resources/sentry-template.peb#L2) has event_id if defined, as well as in the [sentry.peb](https://github.com/kestra-io/plugin-notifications/blob/ac4a2f8d18b873c06a764989fa904b9ebc38a7eb/src/test/resources/sentry.peb#L2). 

I updated the SentryAlertTest with the `eventId` to get the `eventId` defined and used it in the sentry.peb test template, to being able then take it for envelop, otherwise new event Id would be generated
```
        RunContext runContext = runContextFactory.of(Map.of("eventId", UUID.randomUUID().toString().toLowerCase().replace("-", ""),"extra", Map.of(
                "title", "Sentry test alert notification",
                "text", "ge *with some bold text* an",
                "service", IdUtils.create()))
                                                    );
```
Otherwise I should somehow take it from payload string.

Reply:
@tchiotludo do you have any update here or we can resolve this conversation ?